grammar.js

```diff
@@ -723,7 +723,7 @@ module.exports = grammar({
     member_expression: $ => prec('member', seq(
       field('object', choice($.expression, $.primary_expression)),
       choice('.', '?.'),
-      field('property', alias($.identifier, $.property_identifier))
+      field('property', alias(choice($.identifier, $.private_identifier), $.property_identifier))
```

Comment:
Maybe we should actually differentiate between private and public properties in the public API. What do you think?

If we do want that, I think we could:
* rename `private_identifier` to `private_property_identifier`
* in `_property_name`, move `private_property_identifier` *outside* the `alias()`
* similarly, change this line to:

```js
field('property', choice(
  $.private_property_identifier,
  alias($.identifier, $.property_identifier),
))
```

Reply:
I think that makes a lot of sense. Updated.